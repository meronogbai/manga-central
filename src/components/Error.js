import styles from '../styles/Error.module.css';

const Error = () => (
  <main className={styles.error}>
    <div>
      <h1>Oops!</h1>
      <p>Seems like you&apos;ve encoutered an error.</p>
      <img src="https://i.kym-cdn.com/photos/images/original/001/401/347/312.jpg" alt="Error Meme" width="561px" height="265px" />
    </div>
  </main>
);

export default Error;
