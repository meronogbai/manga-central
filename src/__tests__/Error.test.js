import { create } from 'react-test-renderer';
import Error from '../components/Error';

test('renders correctly', () => {
  const tree = create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
});
