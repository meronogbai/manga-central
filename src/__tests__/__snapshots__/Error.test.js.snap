// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="error"
>
  <div>
    <h1>
      Oops!
    </h1>
    <p>
      Seems like you've encoutered an error.
    </p>
    <img
      alt="Error Meme"
      src="https://i.kym-cdn.com/photos/images/original/001/401/347/312.jpg"
    />
  </div>
</main>
`;
